src/app/component/upcoming-timeline/upcoming-timeline.component.ts

```diff
@@ -42,12 +42,18 @@ export class UpcomingTimelineComponent implements OnInit {
     let overDue: any = {
       "primaryCategory":[],
       "status":['0','1'],
-      "timeDuration":[3], 
+      "timeDuration":['3sm'], 
       "competencyArea": [], 
       "competencyTheme": [], 
       "competencySubTheme": [], 
       "providers": [] 
     }
     this.filterValueEmit.emit(event === 'upcoming'? upcomingData: overDue)
   }
+  scroll(el: any) {
+    let element = el
+    let topOfElement: any = document.getElementById(element)
```

Comment:
please make sure scroll is working properly for all screen resolution

Reply:
yes its working